incubator/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: "{{ printf "%s-%s" .Release.Name .Values.Master.Name | trunc 24 }}"
```

Comment:
Any reason that we're not using `{{template "fullname" . }}` everywhere? I know the restriction is only on DNS names in svc records but would it be better to have consistency here?